### DISCUSSION TITLE
Librewolf with betterfox

### DISCUSSION THREAD

**Discussion by mufafa9** [yokoffing/Betterfox#339] [Q&A] [answered]
I wanted to ask if i use betterfox with librewolf  will it interfere with or not or should i just use peskyfox smoothfox and fastfox

**Comment by mufafa9** [comment]
I started using brave now🫩

**Comment by yokoffing** [comment, marked as answer]
Apologies for missing this.
Yes, you can use Betterfox with Librewolf. Librewolf is more or less most of Arkenfox built-in and telemetry removed.
